Src/Shared/Core/Logger.cpp

```diff
@@ -8,69 +8,33 @@
 	#include <sys/ioctl.h>
 #endif
 
-void Log(ELogType flags, const std::string& text)
-{
-	Logger::Instance()->Log(flags, std::move(text));
-}
-
-void LogProgress(u64 current, u64 total)
-{
-	Logger::Instance()->LogProgress(current, total);
-}
+using namespace std;
```

Comment:
:(

Reply:
While `using` in headers would run the risk of irreparably polluting other files, I personally see no reason against liberally `using` namespaces within cpp files if there are no conflicts. Anything I am missing?

In some of my other projects I really enjoy the less redundant and more readable code produced by not repeating `std::` et al. hundreds of times.

Reply:
@Tom94 is generally speaking considered bad practice http://stackoverflow.com/a/1453605/1934465

Reply:
Whether or not it's "generally speaking considered bad practice" really shouldn't be an argument on it's own. That said, the SO answer you linked contains some good arguments against it that I wasn't fully aware of to that extent. Will revert this particular change since most of the codebase doesn't `using` anyway 👍